app/models/pageflow/application_query.rb

```diff
@@ -0,0 +1,12 @@
+module Pageflow
+  class ApplicationQuery
+    class Scope
+      protected
+
+      # @api protected
```

Comment:
`@api protected` is not a thing so far. Let's make `ApplicationQuery` `@api private` completely since we do not expect anyone to inherit from it.